<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmCityHelper</class>
 <widget class="QWidget" name="frmCityHelper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QListWidget" name="listProvince">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>39</y>
     <width>191</width>
     <height>550</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listCity">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>39</y>
     <width>191</width>
     <height>550</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listCounty">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>39</y>
     <width>191</width>
     <height>550</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>591</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="labProvince">
      <property name="text">
       <string>省份</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cboxProvince">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labCity">
      <property name="text">
       <string>市区</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cboxCity">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labCounty">
      <property name="text">
       <string>县城</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cboxCounty">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
